app/services/record_declarations/base.rb

```diff
@@ -55,6 +60,19 @@ def initialize(params:)
       self.declaration_type = params[:declaration_type]
     end
 
+    def set_eligibility
+      if participant_declaration.duplicate_declarations.count > 1
+        raise MultipleParticipantDeclarationDuplicate
```

Comment:
This could occur if we had a teacher registering with 3 different email address. In which we might want to take a closer look. This was a mean to sort of red flag something is probably a bit off. 
I'm happy to remove this check, I agree that is a bit defensive.

Reply:
Yeah, I think we should remove it. It seems to me like we might care about this, but it's not a show stopper. As long as we make subsequent declarations ineligible.